src/net_processing.cpp

```diff
@@ -1678,8 +1680,11 @@ bool PeerManagerImpl::HasAllDesirableServiceFlags(ServiceFlags services) const
 
 ServiceFlags PeerManagerImpl::GetDesirableServiceFlags(ServiceFlags services) const
 {
-    if (services & NODE_NETWORK_LIMITED && GetServicesFlagsIBDCache()) {
```

Comment:
6ed53602ac7c565273b5722de167cb2569a0e381

if you touch it again, i suggest removing `GetServicesFlagsIBDCache` definition here rather than in the last commit. At this point it's already unused. Saves some review time.